deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -3040,6 +3051,15 @@ spec:
           verbs:
           - use
         serviceAccountName: rook-ceph-system
+      - rules:
```

Comment:
Any reason on why these are added twice?

Reply:
- This came because the role was created from code and yaml file as well. I have removed the code part which i have also added in ocsinitialization controller